src/app/users/users.component.html

```diff
@@ -1 +1,20 @@
-<p>users works!</p>
+<br />
+<div *ngFor="let u of users" class="card" style="width: 18rem">
+  <img
```

Comment:
remove this img tag
if the backend does not store user images, Why you include this img here ?

Reply:
It is a "logo" user image... Thought it was elegant..

Reply:
done!